### DISCUSSION TITLE
Opening the Discussions page!

### DISCUSSION THREAD

**Discussion by Raytwo** [Raytwo/Cobalt#32] [Announcements]
Hey there,

In an effort to centralize how information is conveyed to the user base (especially mod makers), we'll start using the Discussions page on this repository to communicate important (breaking or not) changes to Cobalt when necessary.

I am aware this isn't the most visible of places and will still communicate to people directly when possible, but expect reminders and links to important announcements in release notes when relevant.

You may have noticed that we have different categories, which are mostly the default ones for the time being.

Feel free to use the ``Issues`` category when running into a problem before officially reporting, so other people can eventually confirm that you're not alone in this case.

The ``Poll`` category will probably not be used often, but we might use it to decide on what feature takes priority on our roadmap when needed.

As for ``Suggestions``, as the name implies, you may suggest features or quality-of-life changes there if you so desire.

Keep in mind that not getting a reply from us does not mean we are not reading, we'd just rather not shoot ourselves in the foot by agreeing with an idea only to realize it would be too complicated.

**Comment by Deshuri** [comment]
is there tutorial video install cobalt in ryujinx?
I am very suck at english to understand how to install cobalt mods :(

**Comment by oioioioioifan** [comment]
i have tried for like half an hour to install this but i can't do it, i have no idea what the release.zip is, could someone please respond and tell me what that is? (pd: i agree with the other comment about a video tutorial, i'm too bad at this and tend to need a visual guide)
